MediaGalleryUi/Test/Mftf/ActionGroup/AdminEnhancedMediaGalleryVerifyImageDetailsWithDescriptionActionGroup.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminEnhancedMediaGalleryVerifyImageDetailsWithDescriptionActionGroup">
```

Comment:
Let's have the action group verifying only the description and use it in combination with existing `AdminEnhancedMediaGalleryVerifyImageDetailsActionGroup`
```suggestion
    <actionGroup name="AdminEnhancedMediaGalleryVerifyImageDescriptionActionGroup">
```